src/pages/profile-page/profile-page.styles.js

```diff
@@ -72,55 +78,38 @@ export const useStyles = makeStyles(({ palette }) => ({
     }
   },
   button: {
-    fontSize: '1rem',
-    lineHeight: '20px',
-    width: '100%',
     gridColumn: 'span 3',
-    textTransform: 'uppercase'
+    height: '48px'
   },
   userActions: {
     display: 'flex',
     flexDirection: 'column',
     justifyContent: 'flex-start',
-    width: 445,
-    '@media (max-width: 1200px)': {
+    justifySelf: 'center',
+    maxWidth: 635,
+    width: '100%',
```

Comment:
Width of grid children should be controlled by grid if they should have fullWidth

Reply:
fixed